README.md

```diff
@@ -266,6 +267,11 @@ The Asana section ID in the Asana Project
 Comma-separated IDs of Asana tags to be added to the task i.e. https://app.asana.com/0/1208613272217946/
 ### `asana-collaborators`
 Comma-separated Asana user IDs to be added as collaborators to the task
+### `asana-assignee`
+GID of user to assign the task to
+### `asana-task-custom-fields`
+Asana task custom fields hash, encoded as a JSON string i.e. '{"XXXXX":"YYYYY"}'
```

Comment:
Maybe let's add a link to how one can learn these values on a per-project basis? Can use https://app.asana.com/api/1.0/projects/69071770703008 as an example.